packages/example-getting-started/docs/5-datasource.md

```diff
@@ -16,7 +26,9 @@ be using the memory connector provided with the Juggler.
 ```
 
 Create another folder called `datasources` in the `src` directory, and inside
-that folder, create a new file called `db.datasource.ts`.
+that folder, create a new file called `db.datasource.ts`. This file will create
+a strongly-typed export of our datasource using the `DataSourceConstructor`,
```

Comment:
I guess it was because you'd have to recompile your TypeScript (or hack the JS manually) to change it if it was a part of the ts file. This way, it's just a drop-in replacement of the datasource config.

I should probably still change it for the tutorial, shouldn't I? :P

Reply:
I think we should make the code and docs changes in a separate PR.